docker/docker/src/main/java/org/arquillian/cube/docker/impl/client/CubeDockerConfiguration.java

```diff
@@ -65,6 +67,7 @@
     private boolean removeVolumes = true;
     private AutoStartParser autoStartContainers = null;
     private DockerAutoStartOrder dockerAutoStartOrder = null;
+    private boolean cleanBuildImage = true;
```

Comment:
Cosmetic thing but I would move it next to `removeVolumes`. Somehow logically belongs there.